wikimedia/wikipedia-ios
# Native Editor - Allow full talk page editing

**Phabricator:** 
https://phabricator.wikimedia.org/T331936

### Notes
This PR allows editing of talk pages with the native editor. It adds an entry point in the overflow menu and adjusts the editing flow.

 Note: This PR is stacked onto #4721. Don't merge this until that is merged. 

### Test Steps
1. On Staging, go to a talk page on Test Wikipedia.
2. From overflow menu, tap Edit Source
3. After editing in native editor, tap Next.
4. You should skip the preview and see the "Save changes" view.
5. Tap the new "Show web preview" button from the "Save changes" view.
6. You should see the preview view pushed on, with the new toast "All content is shown in simplified format". There should be no Next button.
7. Tap back, then publish your changes.
8. Confirm you see a success toast and your changes are reflected on the talk page.
